index.html

```diff
@@ -24,6 +24,10 @@
             <i id="line2"></i>
             <i id="line3"></i>
         </div>
+        <span class="searchbar-wrap">
+            <input type="text" placeholder="search github" id="searchbar-input"/>
+            <a href="" target="_blank" id="searchbar-a">🔎</a>
+        </span>
```

Comment:
Why don't us doing this with a simple form:

```html
<form method="get" action="https://github.com/pixijs/examples/search">
  <input type="text" name="q" placeholder="Search GitHub...">
  <button type="submit">Search</button>
</form>
```

Then you don't need any JavaScript.